gulpfile.js

```diff
@@ -45,7 +45,7 @@ gulp.task('test', function() {
   return gulp.src(paths.js)
     .pipe(istanbul())
     .pipe(istanbul.hookRequire())
-    .on('finish', function () {
+    .on('finish', function() {
```

Comment:
This change is unrelated. If you want to add some linting for this, i'm game, but do it in another PR please.